applications/models.py

```diff
@@ -71,6 +71,21 @@ def __str__(self):
 
 
 class Organisation(models.Model):
+    COMPANY = 1
+    REGISTERED_ASSOCIATION = 2
+    PUBLIC_ASSOCIATION = 3
+    UNREGISTERED_ASSOCIATION = 4
+    MUNICIPALITY_CONSORTIUM = 5
+    RELIGIOUS_COMMUNITY = 6
+
+    TYPE_CHOICES = [
+        (COMPANY, _("Yritys")),
```

Comment:
I think translated strings should always be in English in code and be translated in translation files to other languages.